orchestration/dagster_orchestration/hca_orchestration/tests/test_resources.py

```diff
@@ -19,13 +19,13 @@
 @contextmanager
 def initialize_resource(resource_def: ResourceDefinition, config: DagsterConfigDict = {}):
     with build_resources(
-        {
-            'test_resource': resource_def,
-        },
-        DagsterInstance.get(),
-        {
-            'test_resource': config
-        }
+            {
+                'test_resource': resource_def,
+            },
+            DagsterInstance.get(),
+            {
+                'test_resource': config
+            }
```

Comment:
very unimportant: why the indent change?

Reply:
Autopep8 commit hook fired, not sure why it picked that up.